### DISCUSSION TITLE
Is there a way to use MAPE instead of MAE?

### DISCUSSION THREAD

**Discussion by aegek** [ourownstory/neural_prophet#212] [Q&A - get help using NeuralProphet]
I was searching for it but couldn't find anything related

**Comment by cbergmeir** [comment]
Hi, you mean using MAPE as a loss function to optimise? That tends to not work very well.

**Reply by aegek** [reply]
Yes I was, doesn't it differ from dataset to dataset?

**Comment by ourownstory** [comment]
@aegek you can use any loss function defined in `torch.nn.modules.loss._Loss`
You can even define your own, inheriting from  `torch.nn.modules.loss._Loss`.

If you simply want to record MAPE as a metric, this will be supported when  #185 gets merged.
